CRM/Utils/System.php

```diff
@@ -452,11 +452,11 @@ public static function crmURL($params) {
 
     return self::url(
       $p,
-      CRM_Utils_Array::value('q', $params),
-      CRM_Utils_Array::value('a', $params, FALSE),
-      CRM_Utils_Array::value('f', $params),
-      CRM_Utils_Array::value('h', $params, TRUE),
-      CRM_Utils_Array::value('fe', $params, FALSE),
+      $params['q'] ?? NULL,
+      $params['a'] ?? FALSE,
+      $params['f'] ?? NULL,
+      $params['h'] ?? TRUE,
+      $params['fe'] ?? FALSE,
       CRM_Utils_Array::value('fb', $params, FALSE)
```

Comment:
I think you could convert this last one too.

Reply:
Thanks, done.